Source/WebCore/bindings/js/JSDOMMapLike.cpp

```diff
@@ -54,11 +54,11 @@ std::pair<bool, std::reference_wrapper<JSC::JSObject>> getBackingMap(JSC::JSGlob
 void clearBackingMap(JSC::JSGlobalObject& lexicalGlobalObject, JSC::JSObject& backingMap)
 {
     auto& vm = JSC::getVM(&lexicalGlobalObject);
-    auto function = backingMap.get(&lexicalGlobalObject, vm.propertyNames->clear);
+    auto function = lexicalGlobalObject.mapPrototype()->getDirect(vm, vm.propertyNames->builtinNames().clearPrivateName());
```

Comment:
How about storing these functions in JSGlobalObject's field? Some of the functions are initialized and stored in JSGlobalObject, and used in prototype definition. We can make this kind of initialization ordering graph via LazyProperty. For example, we can check `JSGlobalObject::arrayProtoToStringFunction()`.